generate.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# This should be changed to different github link that points to our particular project
+# #egg=[] should be specified in order to run.
+#Example: pipenv install -e git+https://github.com/requests/requests.git#egg=requests
+#pipenv install -e git+https://github.com/faraonc/hwsc-api-blocks@master#egg=hwsc_transaction_svc_pb
+#git sparse checkout
+#how to make a repo pipenv
+
+pipenv install grpcio-tools
+pipenv install googleapis-common-protos
+pipenv install requests
```

Comment:
Move the lines starting from https://github.com/faraonc/hwsc-file-transaction-svc#1-upload until the end of the file https://github.com/faraonc/hwsc-file-transaction-svc#3-delete

to a new file called memo.txt. These lines should not be in the README.md. README.md should not contain algorithms.